lib/game.rb

```diff
@@ -1,13 +1,20 @@
+require_relative 'board'
+require_relative 'console'
 class Game
+  def initialize(board, console)
+    @board = board
+    @console = console
+  end
+
   def welcome
-    puts 'Hello, welcome to Tic Tac Toe.'
+    @console.print_message('Welcome to Tic Tac Toe')
   end
 
-  def display_board
-    puts '   |   |   '
-    puts '-----------'
-    puts '   |   |   '
-    puts '-----------'
-    puts '   |   |   '
+  def play
+    @console.print_message(@board.board_string)
```

Comment:
What do you think about the way `@board.board_string` reads?

Reply:
It sounds redundant, but I was trying to remain descriptive within the sole context of the board class.